<?xml version='1.0'?>

<launch>

    <!--    start the server   -->
    <node name="sara_move_arm_server" pkg="wm_moveit_server" type="move_arm_server"/>

</launch>
